docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,235 @@ The following table shows how risk levels are applied to a host, based on the no
+image::images/hrs-all-hosts.png[Host risk score on the Hosts page]
 
-To go to the host's detail page, left-click any host's corresponding bar in the histogram, then select *Go to Host View*.
+The *Hosts by risk* tab on the Hosts page:
 
 [role="screenshot"]
-image::images/go-to-host-view.png[]
+image::images/hosts-by-risk-tab.png[Host risk score on the host details page]
 
-The data tables beneath the histogram display associated rules, users, and MITRE ATT&CK tactics of risky hosts. The table data is sorted in reverse chronological order by default, with the highest total risk score at the top. Use this information to triage alerts with the highest risk in your network.
+The Overview section on the host details page:
 
 [role="screenshot"]
-image::images/data-tables.png[]
+image::images/hrs-overview-section.png[Host risk score in Overview section]
 
-[discrete]
-==== Additional places to visualize host risk score data
+The *Hosts by risk* tab on the host details page:
 
-You can also visualize host risk score data in the following places in the {security-app}:
+[role="screenshot"]
+image::images/hosts-by-risk-details-page.png[Host risk score on the Hosts by risk tab]
 
-The *Overview* tab on the Alert details flyout:
+You can also visualize host risk score data using prebuilt dashboards that are automatically imported once the feature is enabled. 
+
+To access the dashboards:
+
+. In {kib}, go to *Analytics* -> *Dashboard*, then search for `risk score`.
+. Select *Drilldown of Host Risk Score* to drill down further into risk components of a particular host, or *Current Risk Score for Hosts* to display a list of current risky hosts in your environment.
 
 [role="screenshot"]
-image::images/score-in-flyout.png[]
+image::images/select-hrs-dashboard.png[Select host risk score dashboard]
 
-The *Host risk classification* column in the All hosts table on the Hosts page:
+In this example, we'll explore the *Drilldown of Host Risk Score* dashboard.
 
 [role="screenshot"]
-image::images/hrs-all-hosts.png[]
+image::images/full-dashboard.png[Shows dashboard]
 
-The *Hosts by risk* tab on the Hosts page:
+Use the histogram to track how the risk score for a particular host has changed over time. To specify a date range, use the date and time picker, or drag and select a time range within the histogram.
+
+[role="screenshot"]
+image::images/histogram.png[]
+
+To go to the host's details page, left-click any host's corresponding bar in the histogram, then select *Go to Host View*.
 
 [role="screenshot"]
-image::images/hosts-by-risk-tab.png[]
+image::images/go-to-host-view.png[]
 
-The *Overview* section on the Host details page:
+The data tables beneath the histogram display associated rules, users, and MITRE ATT&CK tactics seen on risky hosts. The table data is sorted in reverse chronological order by default, with the highest total risk score at the top. Use this information to triage hosts with the highest risk in your network.
```

Comment:
```suggestion
The data tables beneath the histogram display associated rules, users, and MITRE ATT&CK tactics seen on risky hosts. By default, the tables are sorted by risk, with the highest total risk scores at the top. Use this information to triage your highest risk hosts.
```
Maybe I'm missing something, but in the screenshots I'm not seeing anything chronological.